src/content/admin/resource-manager.mdx

```diff
@@ -92,6 +120,21 @@ The Resource Manager relies on the Kubernetes Metric Server. If the Metric Serve
 
 ### Known Limitation: Support for ReplicaSet and CronJob
 
-The Resource Manager does not natively support optimizing ReplicaSets and CronJobs due to limitations in differentiating between them and their controllers (e.g., Deployments and Jobs). By default, the Resource Manager infers the original controller’s name from the pod’s name (e.g., extracting `${deploymentName}` from a pod named `${deploymentName}-644d69dbc4-fv5g5`).
```

Comment:
I think we documented too much internal concepts that are not useful o the user, I've simplified and added an example. Let me know if you think we should explain more the technical reasons, and I can find a way to put it back.